model_compression_toolkit/exporter/target_platform_export/keras/exporters/fakely_quant_keras_exporter.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import keras.models
+import tensorflow as tf
+import tensorflow_model_optimization as tfmot
+from keras.engine.base_layer import Layer
+from keras.engine.input_layer import InputLayer
+from tensorflow_model_optimization.python.core.quantization.keras.default_8bit.default_8bit_quantize_configs import \
+    NoOpQuantizeConfig
+
+from model_compression_toolkit.core.common import Logger
+from model_compression_toolkit.exporter.fully_quantized.keras.builder.quantize_config_to_node import \
+    SUPPORTED_QUANTIZATION_CONFIG
+from model_compression_toolkit.exporter.fully_quantized.keras.quantize_configs.weights_quantize_config import \
+    WeightsQuantizeConfig
+from model_compression_toolkit.exporter.target_platform_export.keras.exporters.base_keras_exporter import BaseKerasExporter
+
+
+class FakelyQuantKerasExporter(BaseKerasExporter):
+    """
+    Exporter for fakely-quant Keras models.
```

Comment:
Consider extend the description for the exporter classes (also in Int8 exporter even though it is not implemented yet).
The terminology is not necessarily clear, maybe add something like "In a fakely-quant model all quantized tensors values are still saved in float32, but are quantized to a set of finite values according to the chosen quantization grid..."
(not necessarily this phrasing but the general idea...)